### PR TITLE
Add Support for User Relation

### PR DESCRIPTION
**Pull request by mattwoolnough**
https://github.com/scrthq/PSGSuite/issues/129

### CODE REVIEW

**Comment by scrthq** [comment]
Thanks for sending this over, @mattwoolnough!! Going to check it out ASAP, busy week this week!!

File: PSGSuite/Public/Users/New-GSUser.ps1 [hunk 1]
```diff
@@ -194,7 +202,7 @@ function New-GSUser {
                 switch ($prop) {
                     PrimaryEmail {
                         if ($PSBoundParameters[$prop] -notlike "*@*.*") {
-                            $PSBoundParameters[$prop] = "$($PSBoundParameters[$prop])@$($Script:PSGSuite.Domain)"
+                            $PSBoundParameters[$prop] = "$($PSBoundParameters[$prop])@$($global:PSGSuite.Domain)"
```
[the remark this answers is not in the document]
Comment (scrthq) [reply PSGSuite/Public/Users/New-GSUser.ps1]: Hey @mattwoolnough - Can you revert `$global:PSGSuite.Domain` back to `$script:PSGSuite.Domain`? Just sanity checked and everything throughout the module for that variable is set at the `Script` scope (a.k.a. "module" scope for this use case), not the `Global` scope.

Everything else looks good!

**Comment by scrthq** [comment]
Merging this in, will get it deployed by this weekend once I add a few other pending items to the release! I'll comment on the original issue #129 once the additions are deployed to the PowerShell Gallery.

Thanks again, @mattwoolnough!!

**Comment by scrthq** [comment]
@mattwoolnough gotcha! That's awesome! If you feel there's value in using a Global scoped variable, I'm all ears. Happy to help solve a problem if it means less workarounds needed on your, or anyone else's, end!

**Comment by mattwoolnough** [comment]
I don't think that my solution was the most elegant one, but it got me where I needed to be. 

The issue is that there is no local file system from which to read the config file.

**Comment by scrthq** [comment]
Interesting! How are you solving the lack of a file system? This would be a scenario I'd love to support, if possible!

**Pull request merged**